docs/develop/api-reference/dictionary-collections.md

```diff
@@ -0,0 +1,201 @@
+---
+sidebar_position: 1
+sidebar_label: Dictionary collections
+title: Dictionary collection API reference
+description: Learn how to interact with the API for dictionary collection data types in Momento Serverless Cache.
+slug: /develop/api-reference/collections/dictionary
+---
+
+# Dictionary collection API reference
+
+## Dictionary methods
+This page details the Momento API methods for [dictionary collections](./../datatypes.md#dictionary-collections).
+
+### DictionaryFetch
+
+Gets a dictionary item from a cache.
+
+| Name            | Type   | Description                                   |
+| --------------- | ------ | --------------------------------------------- |
+| cacheName       | String | Name of the cache.                            |
+| dictionaryName  | String | The name of the dictionary to be retreived. |
+
+<details>
+  <summary>Method response object</summary>
+
+The response object for DictionaryFetch returns three possible options, a cache hit, miss, or an error.
+
+* Cache hit
+    - `valueDictionaryBytesBytes()`: Map<Bytes, Bytes>
+    - `valueDictionaryStringString()`: Map<String, String>
+    - `valueDictionaryStringBytes()`: Map<String, Bytes>
+    - `valueDictionaryBytesString()`: Map<Bytes, String>
+    - `toString()`: string - displays the key/value pairs, truncated.
+* Cache miss
+* Cache error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+### DictionaryGetField
+Get one field from a dictionary item in the cache.
+
+| Name            | Type         | Description                                   |
+| --------------- | ------------ | --------------------------------------------- |
+| cacheName       | String       | Name of the cache.                            |
+| dictionaryName  | String       | Name of the dictionary to be retreived. |
+| field           | String/bytes | Name of the field in the dictionary to be retreived. |
+
+<details>
+  <summary>Method response object</summary>
+
+* Cache hit
+    - `fieldString()`: string
+    - `fieldBytes()`: bytes
```

Comment:
not sure if it might be worth stating that these two return the field name?  if you feel like that is obvious I'm good

Reply:
When you say they return the field name, doesn't this return the entire entity from the dictionary? So the key and value? Maybe I am missing something here.

Reply:
"Field" is the name of the field from the dictionary.
"Value" is the value of the field from the dictionary.